File: TestAppForAbb.Core/Models/Measure.cs
using System;
using System.Collections.Generic;

namespace TestAppForAbb.Core.Models
{
    /// <summary>
    /// Measure base class
    /// </summary>
    public class Measure : AbstractEntity
    {
        /// <summary>
        /// Time of measurement
        /// </summary>
        public DateTime Time { get; set; }

        /// <summary>
        /// Related measures of Current
        /// </summary>
        public virtual ICollection<CurrentMeasure> CurrentMeasures { get; set; }

        /// <summary>
        /// Related measures of Torque
        /// </summary>
        public virtual ICollection<TorqueMeasure> TorqueMeasures { get; set; }

        /// <summary>
        /// Related measures of Pressure
        /// </summary>
        public virtual ICollection<PressureMeasure> PressureMeasures { get; set; }
    }
}

File: TestAppForAbb/ClientApp/src/app/models/measure.ts
export class Measure {
    constructor(
        public time: Date,
        public motorName: string,
        public current: number | null,
        public currentDifference: number | null,
        public torque: number | null,
        public torqueDifference: number | null,
        public pressure: number | null,
        public pressureDifference: number | null
    ) { }
}